### ISSUE TITLE
Graphics a bit off

### ISSUE THREAD

**Issue by Remko76** [reptilex/tesla-style-solar-power-card#103]
Today I added my "grid_to_battery" entity and I see the corresponding line in the graphics is not quite aligned (see attachement).

I know this is a very, very minor thing but my OCD has an issue with this...;
![Capture](https://user-images.githubusercontent.com/75678314/152147094-5a1f5601-6170-42b8-83de-e302a8fe4554.PNG)
)

**Comment by AviadorLP** [comment]
Mine also ;)

**Comment by neildsb** [comment]
+1

**Comment by reptilex** [comment]
Yes I never tested the grid to battery since I don't have that functionality with my inverter, but I will look into it.

**Comment by reptilex** [comment]
Think I repaired this, if not I would need a few more details about the size of your card in pixels and your data.
